### ISSUE TITLE
Implementation Questions Regarding BLIP Version of FDCA Framework

### ISSUE THREAD

**Issue by weiwei-1113** [May2333/FDCA#5]
Hello! First of all, thank you very much for your excellent work in the field of video-based composed retrieval. The FDCA approach is very inspiring!

I encountered a performance gap when trying to reproduce the BLIP version results mentioned in your paper:

- Paper reported results: R@1: 26.79, R@5: 63.21, R@10: 78.65, R@50: 97.25

- My reproduction results: R@1: 19.28, R@5: 56.18, R@10: 74.19, R@50: 96.61

Since there is a noticeable gap in R@1, R@5 and R@10, I would like to ask whether you plan to share the implementation code for the BLIP version in the future? 

Thank you!

**Comment by May2333** [comment]
Apologies for the delayed reply because I've been quite busy lately :(

Regarding the performance difference you observed with BLIP, one possible reason could be the dropout setting. In our previous work, we found that CLIP tended to overfit on our dataset, so we used a dropout rate of 0.5 to mitigate this. However, a dropout rate of 0.5 appears too high for BLIP in our experiments. Therefore, we set the dropout rate to 0.1 for BLIP. Could you please check if this difference in configuration explains the performance variance?

Additionally, we have uploaded the code for the BLIP version and re-ran the training and testing today. The results are shown in the figure below. Please note that the tests were run on a different machine (moving from a 3090 to an A100), which resulted in slight differences in the results, though the overall trend remains consistent.

<img width="1605" height="797" alt="Image" src="https://github.com/user-attachments/assets/6ec0b0f3-2670-4019-bded-4da7c93fba08" />
